collector/pg_wal.go

```diff
@@ -66,19 +66,34 @@ func (c PGWALCollector) Update(ctx context.Context, instance *instance, ch chan<
 		pgWALQuery,
 	)
 
-	var segments uint64
-	var size uint64
+	var segments sql.NullInt64
+	var size sql.NullInt64
 	err := row.Scan(&segments, &size)
 	if err != nil {
 		return err
 	}
+
+	var segmentsValue float64
```

Comment:
segments should never be NULL. It's a `count(*)` which I don't believe can return a NULL value. The error you reported was on the size column. If Aurora is returning NULL for that column, I think there should not be a metric emitted from the exporter. I don't know what value a metric for WAL size with a value of 0 is.